apidoc/Titanium/UI/Window.yml

```diff
@@ -441,24 +441,25 @@ properties:
 
   - name: extendSafeArea
     summary: |
-        Specifies whether the content (subviews) of the window  will render inside the safe-area or not.
-        Only used in iOS 11.0 and later.
+        Specifies whether the screen insets/notches are allowed to overlap the window's content or not.
     description:  |
-        If set `true` (the default), then the content of the window will be extended to fill the whole screen and
-        under the system's UI elements (such as the top status-bar) and physical obstructions (such as the iPhone X
-        rounded corners and top sensor housing). In this case, it is the app developer's responsibility to position
-        views so that they're unobstructed.
+        If set `true`, then the contents of the window will be extended to fill the whole screen and allow the
+        system's UI elements (such as a translucent status-bar) and physical obstructions (such as the iPhone X
+        rounded corners and top sensor housing) to overlap the window's content. In this case, it is the app
+        developer's responsibility to position views so that they're unobstructed. On Android, you can use the
+        [Window.safeAreaPadding](Titanium.UI.Window.safeAreaPadding) property after the window has been opened to
+        layout your content within the insets.
 
         If set `false`, then the window's content will be laid out within the safe-area and its child views will be
         unobstructed. For example, you will not need to position a view below the top status-bar.
 
         Read more about the safe-area layout-guide in the [Human Interface Guidelines](https://developer.apple.com/ios/human-interface-guidelines/overview/iphone-x/).
-    platforms: [iphone, ipad]
+    platforms: [android, iphone, ipad]
     type: Boolean
-    default: true
-    since: 6.3.0
+    default: `true` on iOS. `false` on Android.
```

Comment:
This is causing the docgen validation to error out as it's a string (which fails the parsing of Window.yml, throwing all the other parse errors)
It's super easy to allow the validation script to allow us to set `{android: false, ios: true}` here, but I have no idea how this then plays into the generation of the various templates (html, jdsuck, typescript etc.). @jawa9000 do you know how I can check the output?

Reply:
Thanks. I'll give it a go.